replicated-security/duality-testnet-1/join-rs-duality-testnet-1-cv.sh

```diff
@@ -12,16 +12,16 @@ set -ex
 PRIV_VALIDATOR_KEY_FILE=~/priv_validator_key.json
 NODE_KEY_FILE=~/node_key.json
 NODE_HOME=~/.duality
-NODE_MONIKER=duality-second-rehearsal-1
+NODE_MONIKER=duality-testnet-1
 SERVICE_NAME=cv-duality
 # ***
 
 CHAIN_BINARY='dualityd'
-CHAIN_ID=duality-second-rehearsal-1
-SEEDS="cebd80328abca02408d1b7199f9b1a835a14aa31@p2p.testnet-1.duality.xyz:26656"
+CHAIN_ID=duality-testnet-1
```

Comment:
This doesn't match the genesis file chain ID

Reply:
Done